cmd/commands/all.go

```diff
@@ -47,6 +47,10 @@ func setConfigFromFlags(cmd *cobra.Command, conf config.KubeauditConfig) config.
 		conf.AuditorConfig.Capabilities.AllowAddList = capabilitiesConfig.AllowAddList
 	}
 
+	if flagset.Changed("paths") {
```

Comment:
Good catch, I forgot about this! Can we make `"paths"` into a constant though like `capsAddFlagName`?